plugins/chat/assets/javascripts/discourse/components/chat-drawer.js

```diff
@@ -156,31 +176,26 @@ export default Component.extend({
   },
 
   _checkSize() {
-    if (!this.chatStateManager.isDrawerActive) {
-      return;
-    }
-
     this.sizeTimer = throttle(this, this._performCheckSize, 150);
   },
```

Comment:
This function is throwing an error when you resize the window now.

Also, not sure if related, but there is an element that's periodically overflowing.

https://user-images.githubusercontent.com/66427541/216617041-2539cada-6447-47fd-bae7-381c2ede8ce1.mp4

Reply:
Good catch thanks. I fixed it at some point and then thought it was gone and removed @bind, and somehow didn't see the error anymore 🤔 BUt apparently it was still there.